software/apt-lib.pl

```diff
@@ -457,7 +457,34 @@ sub list_package_repos
 foreach my $f ($sources_list_file, glob("$sources_list_dir/*")) {
 	my $lref = &read_file_lines($f, 1);
 	my $lnum = 0;
+	my (%repo, @types);
+	my $repo = sub {
+		foreach my $type (@types) {
+			my @suites = @{$repo{'suites'}};
+			foreach my $suite (@suites) {
+				my @comps = @{$repo{'comps'}};
+				foreach my $comp (@comps) {
+					my $repo =
+					  {
+					    'cannot' => 1,
+					    'file' => $f,
+					    # 'line' => $lnum,
```

Comment:
Why comment out this line?

Reply:
I wanted to point out that line numbers won't function as usual here because we are dealing with a block of interconnected values.

Reply:
I don't understand ... is $lnum not valid? Leaving this commented out is very confusing, either set the `line` field properly or not at all if we don't need it.

Reply:
> I don't understand ... is $lnum not valid? Leaving this commented out is very confusing, either set the `line` field properly or not at all if we don't need it.

Yes, I have removed it! I just wanted to point this out!